### ISSUE TITLE
Additional binder support needed

### ISSUE THREAD

**Issue by vinothinikannaiyan** [asyncapi/java-spring-cloud-stream-template#340]
Based on the binder parameter description,  looks like generator supports one of kafka, rabbit or solace. Default: kafka.

What is the plan for additional binders?  any roadmap for supporting  azure service bus and apache activemq artemis binders ? Looking for the updates. Please share your insights.

**Comment by Mrc0113** [comment]
FYI - complete list of Spring Cloud Stream Binders is here: https://docs.spring.io/spring-cloud-stream/docs/current/reference/html/binders.html#binders

**Comment by vinothinikannaiyan** [comment]
> FYI - complete list of Spring Cloud Stream Binders is here: https://docs.spring.io/spring-cloud-stream/docs/current/reference/html/binders.html#binders

Yes got it. But this list is for using spring cloud stream library. I am looking for the binder codegen support from asyncapi. Do you know asynapi generates code for service bus ? Artemis is not listed by the way

**Comment by Mrc0113** [comment]
Hi @vinothinikannaiyan, 
I was just referencing the list of Spring Cloud Stream binders here b/c the code generator template in this repository ONLY generates Spring Cloud Stream apps and the list of binders supported by the spring cloud stream framework are the only ones we could add support for here.

**Comment by vinothinikannaiyan** [comment]
> Hi @vinothinikannaiyan, I was just referencing the list of Spring Cloud Stream binders here b/c the code generator template in this repository ONLY generates Spring Cloud Stream apps and the list of binders supported by the spring cloud stream framework are the only ones we could add support for here.
> 
> > Artemis is not listed by the way
> 
> If artemis has a cloud stream binder they should open a PR against the Spring repo to get it added. Otherwise nobody will know it exists!
> 
> > Do you know asynapi generates code for service bus
> 
> I would suggest asking in the AsyncAPI slack, but not that I'm aware of.

Thanks for your inputs.
